test/functional/plugin/shada_spec.lua

```diff
@@ -184,7 +154,7 @@ describe('In autoload/shada.vim', function()
         '  + b                 2',
         '  + c    column       3',
         '  + d                 4',
-      }, {{type=1, timestamp=0, data=st({a=1, b=2, c=3, d=4})}})
```

Comment:
@ZyX-I can you review this? AFAICT this is not needed for the test(if the purpose is to make it iterate in predictable order), and `__pairs` shouldn't even work in lua 5.1